AndresConfortiGL/JavaBaseProject
# Implemented Fragments to the project

## Description
Fourth PR - Implemented Fragments to the project.

## Story Type
- New feature :white_check_mark:
- Change request
- Bug fix
- Tech

## How Can This Be Tested?
By pressing the button in the middle of the activity, you will see a fragment dialog with the option to input a number.
By pressing accept, the input will be shown on the activity, and if you press it without writing a number, nothing will happen.
By pressing cancel, the fragment will be closed.

## Evidences
| Nexus S | Pixel 3a |
| ------------- | ------------- |
|![nexus](https://user-images.githubusercontent.com/109096226/181662028-09e31bb2-4d53-4ee5-a21d-2218a68932d7.gif)|![pixel](https://user-images.githubusercontent.com/109096226/181662053-17ab8783-eaf9-489b-9b5c-3ccf7d021003.gif)|

## Tested on
The project was tested on Android 8.0 / API 26 using the following devices:

- Nexus S
- Pixel 3a

## Package Structure
| Screenshot |
| ------------- |
|![package](https://user-images.githubusercontent.com/109096226/181661992-25bded08-a23e-4d00-8b74-4187925322b4.png)|
